### DISCUSSION TITLE
Using VS Code Server Web UI as a front-end of VS Code?

### DISCUSSION THREAD

**Discussion by jducaud** [coder/code-server#3121] [Q&A] [answered]
Hello,

I would like to use **VS Code Server in WSL2-Ubuntu** [VSCS@WSL2] as a front-end for **VS Code in Windows 10** [VSC@W10].

The goal is to access, through the VSCS@WSL2 web UI, the whole coding environment of VSC@W10 (meaning Android SDK, Flutter SDK and all the VSC@W10 extensions I have installed). Note that WSL2-Ubuntu and Windows 10 are on the same machine (WSL2-Ubuntu inside Windows 10).

So I installed the `Remote Development` extension package in VSC@W10. No problem.

Then I downloaded:
- the `Remote Development` extension package (VSIX file)
- the `Remote SSH` extension (VSIX file)
- the `Remote Container` extension (VSIX file)

After that, I manually installed in VSCS@WSL2 those 3 VSIX files (`>Extensions: Install from VSIX`)

The `Remote Container` extension installation failed (incompatible with current VS Code Server version) but the 2 others succeeded.

When I tried the `>Remote-SSH: Connect Current Window to Host...` the Chrome Tab address went to `http://127.0.0.1:8080/?ew=true`, embedding VS Code Server Web UI with its status bar purple instead of cyan.

Despite the fact I set in the VSCS@WSL2 `settings.json`:
```
"remote.SSH.showLoginTerminal": true,
"remote.SSH.useLocalServer": false
```
... I got no prompt in the terminal of the VS Code Server to enter the passphrase of the private SSH key.

Maybe VS Code Server does not support those "Remote" extensions...
Maybe this is architecturally impossible...
Maybe I missed something...

**Comment by jsjoeio** [comment, marked as answer]
Yes, we don't currently support the Remote extensions. See #1315

**Reply by jducaud** [reply]
Ah damn! I missed that one (#1315)...
Thank you for your answer

**Reply by jsjoeio** [reply]
Don't worry, it was hard to find even for me ("remote" and "extensions" are almost in all issues 😂). No problem!
